programs/uxd/tests/integration_tests/api/program_uxd/instructions/process_edit_credix_lp_depository.rs

```diff
@@ -0,0 +1,125 @@
+) -> Result<(), program_test_context::ProgramTestError> {
+    // Read state before
+    let credix_lp_depository_before =
+        program_test_context::read_account_anchor::<CredixLpDepository>(
+            program_test_context,
+            &program_keys.credix_lp_depository_keys.depository,
+        )
+        .await?;
+
+    let redeemable_amount_under_management_cap_before =
+        credix_lp_depository_before.redeemable_amount_under_management_cap;
+    let minting_fee_in_bps_before = credix_lp_depository_before.minting_fee_in_bps;
+    let redeeming_fee_in_bps_before = credix_lp_depository_before.redeeming_fee_in_bps;
+    let minting_disabled_before = credix_lp_depository_before.minting_disabled;
+    let profits_beneficiary_collateral_before =
+        credix_lp_depository_before.profits_beneficiary_collateral;
+
+    // Execute IX
+    let accounts = uxd::accounts::EditCredixLpDepository {
+        authority: program_keys.authority.pubkey(),
+        controller: program_keys.controller,
+        depository: program_keys.credix_lp_depository_keys.depository,
+    };
+    let payload = uxd::instruction::EditCredixLpDepository {
+        fields: uxd::instructions::EditCredixLpDepositoryFields {
+            redeemable_amount_under_management_cap,
+            minting_fee_in_bps,
+            redeeming_fee_in_bps,
+            minting_disabled,
+            profits_beneficiary_collateral,
+        },
+    };
+    let instruction = Instruction {
+        program_id: uxd::id(),
+        accounts: accounts.to_account_metas(None),
+        data: payload.data(),
+    };
+    program_test_context::process_instruction_with_signer(
+        program_test_context,
+        instruction,
+        payer,
+        &program_keys.authority,
+    )
+    .await?;
+
+    // Read state after
+    let credix_lp_depository_after =
+        program_test_context::read_account_anchor::<CredixLpDepository>(
+            program_test_context,
+            &program_keys.credix_lp_depository_keys.depository,
+        )
+        .await?;
+
+    let redeemable_amount_under_management_cap_after =
+        credix_lp_depository_after.redeemable_amount_under_management_cap;
+    let minting_fee_in_bps_after = credix_lp_depository_after.minting_fee_in_bps;
+    let redeeming_fee_in_bps_after = credix_lp_depository_after.redeeming_fee_in_bps;
+    let minting_disabled_after = credix_lp_depository_after.minting_disabled;
+    let profits_beneficiary_collateral_after =
+        credix_lp_depository_after.profits_beneficiary_collateral;
+
+    // Check result
+    if redeemable_amount_under_management_cap.is_some() {
+        assert_eq!(
+            redeemable_amount_under_management_cap_after,
+            redeemable_amount_under_management_cap.unwrap()
+        );
+    } else {
+        assert_eq!(
+            redeemable_amount_under_management_cap_after,
+            redeemable_amount_under_management_cap_before
+        );
+    }
```

Comment:
You may use `unwrap_or`:

```suggestion
    assert_eq!(
        redeemable_amount_under_management_cap_after,
        redeemable_amount_under_management_cap
            .unwrap_or(redeemable_amount_under_management_cap_before)
    );
```